### DISCUSSION TITLE
How can I make some routes partially SSG pre-rendered and partially client-rendered, while keeping other routes entirely client-rendered?

### DISCUSSION THREAD

**Discussion by TwT-L** [vikejs/vike#811] [Discussion]
example:
We have "/" and "/about" routes, in which the route layout includes a dynamic component Header with user status, followed by a static marketing page. Meanwhile, we have a "/user" route that is purely client-rendered, also including a dynamic component Header with user status.

And the current "/user" route is already in production, how can I achieve SSG for "/" and "/about"?

**Comment by brillout** [comment]
Have you seen https://vite-plugin-ssr.com/render-modes?
